src/renderers/shared/fiber/ReactFiber.js

```diff
@@ -106,7 +109,7 @@ var createFiber = function(tag : number, key : null | string) : Fiber {
     memoizedInput: null,
     output: null,
 
-    hasPendingChanges: true,
+    pendingWorkPriority: 0,
```

Comment:
Why isn't Synchronous the highest and largest number? `priorityA <= priorityB` sounds to me like B is more important but you have the opposite. Switching it would solve your 0 problem.